### DISCUSSION TITLE
Getting unauthorized on all API requests

### DISCUSSION THREAD

**Discussion by mattrochon** [gorules/zen#220] [Q&A]
All of my api requests are returning unauthorized, even with a freshly generated PAT

![image](https://github.com/user-attachments/assets/c3f2aa1a-a449-4f6b-9992-e8d249ee7044)

**Comment by mattrochon** [comment]
![image](https://github.com/user-attachments/assets/b011fac7-a0a1-420f-930a-d4d78529bccb)

**Comment by ivanmiletic** [comment]
We will take a look.

EDIT: saw that you are using our demo cloud

**Comment by mattrochon** [comment]
Cloud version, demo is just int he api tester as it wont allow cross site requests to my subdomain 'matt-personal'

The curl request uses the correct domain.

**Comment by ivanmiletic** [comment]
Hi @mattrochon, we are not able to reproduce it. Is there anything unusual such as some characters in the pat or something else?

**Comment by stefan-gorules** [comment]
Moving this to discussion, as it's not directly part of open-source, but rather of BRMS.

**Comment by lockhinator** [comment]
I just encountered this issue on an account I created today. Moving back to an older account I was able to make requests to the API with no issue. Not sure if this may be a bug with newly created accounts in the cloud environment.

**Reply by ivanmiletic** [reply]
@lockhinator just created a new org and a new user, can't reproduce with my PAT.

Would you be able to copy the PAT that you are using (and that is not working) and revoke it as well so it is not active. I would like to inspect characters and hashing on your PAT.

**Reply by lockhinator** [reply]
> @lockhinator just created a new org and a new user, can't reproduce with my PAT.
> 
> Would you be able to copy the PAT that you are using (and that is not working) and revoke it as well so it is not active. I would like to inspect characters and hashing on your PAT.

Here is the request that was failing including the PAT. Tokens have been revoked.

```bash
curl -L -X GET 'https://test-org-smarterp.gorules.io/api/projects/316bc65b-0b8d-46e8-b4a9-6a01751f184e/releases' -H 'Accept: application/json' -H 'Authorization: Bearer grl_pat_RFNF5Am4V-wSsgi56qWq_tDS2uoflWdLfCI5eeQpzlXYetsl'
```
